packages/truffle-debugger/lib/session/index.js

```diff
@@ -176,8 +176,12 @@ export default class Session {
     return await this.doneStepping(controller.reset());
   }
 
-  async continueUntilBreakpoint() {
-    return await this.doneStepping(controller.continueUntilBreakpoint());
+  async continueUntilBreakpoint(
+    breakpoints = this.view(controllerSelector.breakpoints)
```

Comment:
I think we can skip the default here since we're setting defaults internally, but we'll need to use a comment or an explicit `breakpoints = undefined` so it's clear that it's an optional arg.

Reply:
OK the thing I was originally going to write here was wrong.  But yeah obviously we don't actually need redundant defaults like I did.

Btw, note that I also did the redundant default thing when I added multiple advancement; do you think I should take that out, have the default live in only one place?

I think you're right it's worth adding a comment at least.

Reply:
There've been a couple instances recently where doubling-down on defaults has come back to bite. (Truffle was providing its own defaults for solc's `evmVersion` and Ganache's `hardfork`)

So now I'm of the mind that it's probably just best practice to rely on the innermost defaults. In this situation, I think it's fine to leave defaulting as a concern of the controller saga, and leave it to everything external to that with allowable undefineds.

Reply:
Got it!  I'll go back and fix up the `advance` function similarly while I'm at it, then.